app/organisation/invite_rest.py

```diff
@@ -3,16 +3,18 @@
     request,
     jsonify,
     current_app)
-from notifications_utils.url_safe_token import generate_token
+from itsdangerous import SignatureExpired, BadData
+from notifications_utils.url_safe_token import check_token, generate_token
 
 from app.config import QueueNames
 from app.dao.invited_org_user_dao import (
     save_invited_org_user,
-    get_invited_org_user,
+    get_invited_org_user as dao_get_invited_org_user,
     get_invited_org_users_for_organisation
 )
 from app.dao.templates_dao import dao_get_template_by_id
-from app.errors import register_errors
+from app.dao.organisation_dao import dao_get_invited_organisation_user
```

Comment:
Could we replace this, with [this](https://github.com/alphagov/notifications-api/blob/a2cc0df5a70b918bad27beea48cff55362c838e5/app/dao/invited_org_user_dao.py#L16), to avoid the name clash?

Reply:
fixed in 68d24e5